### ISSUE TITLE
[BUG]iOS itemFailedToPlayToEndTime not implemented

### ISSUE THREAD

**Issue by bazookon** [jhomlala/betterplayer#286]
**Describe the bug**
[iOS itemFailedToPlayToEndTime not implemented](https://github.com/jhomlala/betterplayer/blob/6105826524e5c25e4d1a4e2db3b30ed64588eeee/ios/Classes/FLTBetterPlayerPlugin.m#L129)

```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[FLTBetterPlayer itemFailedToPlayToEndTime:]: unrecognized selector sent to instance 0x283053210'
*** First throw call stack:
(0x1ac4599d8 0x1c07c2b54 0x1ac369bbc 0x1ac45c01c 0x1ac45df8c 0x1ac3bc370 0x1ac3bc330 0x1ac3bb928 0x1ac3bb320 0x1b39ae9dc 0x1ac091298 0x1ac092280 0x1ac0745b0 0x1ac3d95e0 0x1ac3d3a88 0x1ac3d2ba0 0x1c3110598 0x1aecc23d8 0x1aecc7958 0x102820a04 0x1ac0b1568)
libc++abi.dylib: terminating with uncaught exception of type NSException
* thread #1, queue = 'com.apple.main-thread', stop reason = signal SIGABRT
    frame #0: 0x00000001d833084c libsystem_kernel.dylib`__pthread_kill + 8
libsystem_kernel.dylib`__pthread_kill:
->  0x1d833084c <+8>:  b.lo   0x1d8330868               ; <+36>
    0x1d8330850 <+12>: stp    x29, x30, [sp, #-0x10]!
    0x1d8330854 <+16>: mov    x29, sp
    0x1d8330858 <+20>: bl     0x1d830df5c               ; cerror_nocancel


```

**To Reproduce**
On HLS error call to unrecognized selector

**Expected behavior**
send callback message and should retry source i guess

**Flutter doctor**
```
➜  flutter flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel beta, 1.26.0-17.3.pre, on macOS 11.2 20D64 darwin-arm,
    locale es-419)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.1)
[✓] Connected device (3 available)

• No issues found!
```

**Better Player version**
- Version: 0.0.52

**Smartphone (please complete the following information):**
 - Device: iPhone 8(phisical) && iPhone 12 max pro (emulator)
 - OS: iOS 14.4

**Comment by jhomlala** [comment]
Fixed in 0.0.53.

**Comment by brax10ward** [comment]
@jhomlala We are getting this error running version 0.0.61. It only happens on a device and not on the simulator. We also have videos in a `ListView`.
